tests/realtikvtest/importintotest4/global_sort_test.go

```diff
@@ -274,8 +274,12 @@ func (s *mockGCSSuite) TestGlobalSortRecordedStepSummary() {
 	s.EqualValues(12, sum.PutReqCnt.Load())
 
 	sum = s.getStepSummary(ctx, taskManager, task.ID, proto.ImportStepWriteAndIngest)
-	s.Greater(sum.RowCnt.Load(), int64(0)) // Fixme: why not 10000?
-	s.Greater(sum.Bytes.Load(), int64(0))  // Fixme: unstable
+	s.EqualValues(sum.RowCnt.Load(), 10000)
+	if !kerneltype.IsNextGen() {
+		s.EqualValues(sum.Bytes.Load(), 2622604)
```

Comment:
`Bytes` is the written kv size, why classic only 28 bytes less than nextgen, shouldn't it be `10000 * 4` for the keyspace prefix?